src/chunk/conversion_from_glib.rs

```diff
@@ -24,3 +25,12 @@ impl<'a> From<&'a library::Parameter> for Mode {
         }
     }
 }
+
+impl<'a> From<&'a RustParameter> for Mode {
+    fn from(orig: &'a RustParameter) -> Mode {
+        Mode {
+            typ: orig.typ,
+            transfer: library::Transfer::None, // TODO: fix this
```

Comment:
Where is this impl used exactly?

Reply:
Literally nowhere. I removed it. Good catch!